CMakeLists.txt

```
@@ -456,6 +456,7 @@ if(SDL_SHARED)
   add_library(SDL3-shared SHARED)
   add_library(SDL3::SDL3-shared ALIAS SDL3-shared)
   SDL_AddCommonCompilerFlags(SDL3-shared)
+  target_compile_options(SDL3-shared PRIVATE "$<$<CONFIG:Debug>:-DDEBUG>")
```

There's `target_compile_definitions` for macro's specifically.
```suggestion
  target_compile_definitions(SDL3-shared PRIVATE "$<$<CONFIG:Debug>:DEBUG>")
```